src/kyc.ts

```diff
@@ -23,11 +25,38 @@ export const kycStatusSchema = z.nativeEnum(KycStatus, {
 // When adding new schemas be sure to also update kycSchemasSchema
 export enum KycSchema {
   PersonalDataAndDocuments = 'PersonalDataAndDocuments',
+  PersonalDataAndDocumentsDetailed = 'PersonalDataAndDocumentsDetailed',
 }
+
 export const kycSchemaSchema = z.nativeEnum(KycSchema, {
   description: 'kycSchemaSchema',
 })
 
+export enum IdentificationDocumentType {
+  IDC = 'IDC',
+  PAS = 'PAS',
+  DL = 'DL',
+}
+
+// need [string, ...string[]] types to get zod enums to compile
```

Comment:
i would have preferred using `IdentificationDocumentType[]` for both of these, but zod errored out because it doesn't like if you create empty enums I guess. I welcome other ideas of getting this to work though